pages/api/members/DELETE.ts

```diff
@@ -1,10 +1,20 @@
 import prisma from '@/prisma/prisma';
 
-export default async function deleteHandler(id: string) {
-  const deletedMember = await prisma.libraryMember.delete({
-    where: {
-      id,
-    },
-  });
-  return deletedMember;
+export default async function deleteHandler(id?: string) {
+  let deletedMember;
+  if (id) {
+    try {
+      deletedMember = await prisma.libraryMember.delete({
+        where: {
+          id,
+        },
+      });
+      return deletedMember;
+    } catch {
+      return;
+    }
+  } else {
+    await prisma.libraryMember.deleteMany({});
+    return { message: 'All members were successfully deleted.' };
```

Comment:
This seems a bit dangerous, we may want to refactor to require a body with a list of members to delete so we don't accidentally nuke anything. I'll add this as an issue though, as we're still in dev mode and the rest lg2m!

Reply:
Yeah I was going to ask about that, will make this edit.